Makefile

```diff
@@ -40,7 +40,7 @@ KVM_GO_VERSION ?= $(GO_VERSION:.0=)
 
 INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
 BUILDROOT_BRANCH ?= 2020.02.12
-REGISTRY?=gcr.io/k8s-minikube
+REGISTRY ?=gcr.io/k8s-minikube
```

Comment:
Don't think it matters at all, but since you added the space before, this is the only one that doesn't have a space after `?=` as well.

Reply:
yeah it's purely aesthetic change, good call

Reply:
fixed